src/foam_node.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
/**
 * @license
 * Copyright 2021 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */